next-frontend/src/components/live/LiveResultsTable.tsx

```diff
@@ -0,0 +1,223 @@
+import { useMemo } from "react";
+import _ from "lodash";
+import { components } from "@/types/openapi";
+import events from "@/lib/wca/data/events";
+import { Link, Table } from "@chakra-ui/react";
+import {
+  centisecondsToClockFormat,
+  formatAttemptResult,
+} from "@/lib/wca/wcif/attempts";
+
+const advancingColor = "0, 230, 118";
+
+const customOrderBy = (
+  competitor: components["schemas"]["LiveCompetitor"],
+  resultsByRegistrationId: Record<string, components["schemas"]["LiveResult"]>,
+  sortBy: "best" | "average",
+) => {
+  const competitorResult = resultsByRegistrationId[competitor.id];
+
+  if (!competitorResult) {
+    return 100000000000 + competitor.id;
+  }
+
+  const result = competitorResult[sortBy];
+
+  return result < 0 ? 100000000000 : result;
+};
+
+export const rankingCellStyle = (
+  result: components["schemas"]["LiveResult"],
+) => {
+  if (result?.advancing) {
+    return { backgroundColor: `rgb(${advancingColor})` };
+  }
+
+  if (result?.advancing_questionable) {
+    return { backgroundColor: `rgba(${advancingColor}, 0.5)` };
+  }
+
+  return {};
+};
+
+export const recordTagStyle = (tag: string) => {
+  const styles = {
+    display: "block",
+    lineHeight: 1,
+    padding: "0.3em 0.4em",
+    borderRadius: "4px",
+    fontWeight: 600,
+    fontSize: "0.6em",
+    position: "absolute",
+    top: "0px",
+    right: "0px",
+    transform: "translate(110%, -40%)",
```

Comment:
This is especially shady. Why do you need inline transforms? If you want to move the component around within the row, think about [Float](https://www.chakra-ui.com/docs/components/float).
To the naked eye (haven't spun up your code at all locally) this looks like "Badge + Float + some **minimal** custom styling on the badge"

Reply:
yeah just using the `recordTagBadge` component that I made works fine